tests/ha/remove_node.pm

```diff
@@ -61,7 +62,7 @@ sub run {
     remove_state_join('HOST', $cluster_name, $node_01_host, $node_02_host);
 
     # Both remove and join the second node by its IP address
-    remove_state_join('IP', $cluster_name, $node_01_ip, $node_02_ip);
+    remove_state_join('IP', $cluster_name, $node_01_ip, $node_02_ip) unless is_sle("=12-sp2");
```

Comment:
Single quotes should be more coherent for `is-sle`.

Reply:
You are right, done.